charts/argo-cd/values.yaml

```diff
@@ -139,7 +139,7 @@ configs:
   ## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml
   cm:
     # -- Create the argocd-cm configmap for [declarative setup]
-    create: true
+    create: false
```

Comment:
Is this intentional change?

Reply:
This was an unintentional change. Let me fix it.